test/Vegetation/plant_hydraulics_test.jl

```diff
@@ -1,7 +1,9 @@
 using Test
```

Comment:
Does this file need to change at all?

Reply:
I had brought some changes, but will look through it again as well as bring in changes proposed above.

Reply:
done